hummingbot/core/data_type/limit_order.pyx

```diff
@@ -26,7 +26,7 @@ cdef class LimitOrder:
                 float(order.price),
                 f"{(0 if mid_price == 0 else abs(float(order.price) - mid_price)/mid_price):.2%}",
                 float(order.quantity),
-                ("n/a" if "-" not in order.client_order_id else
+                ("n/a" if "-" or "//" not in order.client_order_id else
```

Comment:
why do we nee do to look for both the dash and `//`?  is one of them sufficient to separate real orders from paper orders?

Reply:
Thanks for the feedback @mifeng you're right `//` would suffice to identify paper trade orders based on the order ID. Just pushed a commit to change it.